src/.bazelrc

```diff
@@ -9,6 +9,12 @@ test --test_output=all
 # Use Our Custom Toolchain
 build --crosstool_top=//cc_toolchain:toolchain
 
+# Add warnings to Thunderbots code only
+build --per_file_copt=//software/.*,//shared/.*,//firmware/.*,//firmware_new/.*@-Wall,-Wextra,-Wno-unused-parameter
```

Comment:
Can we blacklist the anything that matches `@//` and the `external` folder instead of whitelisting our folders?

Reply:
I tried that, --per_file_copt doesn't work on the @zlib//:zlib dependency of protobuf and I don't know why. Even .* (as blacklist or whitelist) won't capture zlib, and of course zlib throws warnings.

Reply:
OK, thanks for the context.